api/cont.js

```diff
@@ -114,7 +114,17 @@
                 continuation
                     .extend({source: sourceFragment}, idx, sourceLength, value)
                     .evalReference();
-            });
+            };
+
+            if (typeof source.length === 'number') {
+                $.each(source, eachCallback);
+            }
+            else {
```

Comment:
@Helen-Mobify could you explain a bit about why this fixes the issue? Why is it that it's better to use the `for..in` when `source.length` is a number? Is Zepto's each function just completely busted on some devices?